rules.md

```diff
@@ -4,6 +4,7 @@
   - Garbage In/Garbage Out ([#26](https://github.com/Benjamin-Lee/deep-rules/issues/26))
   - Normalize your data ([#7](https://github.com/Benjamin-Lee/deep-rules/issues/7))
   - Make sure data is not biased/skewed ([#43](https://github.com/Benjamin-Lee/deep-rules/issues/43))
+  - How to deal with class imbalance
```

Comment:
I am assuming this refers to bias/skew in terms of "unfairness" so I think adding a separate point on class imbalance makes sense (this issue often comes up in e.g., molecular activity analysis where there are lots of inactive molecules assayed and only a handful of actives)